pom.xml

```diff
@@ -19,12 +19,12 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <mysqlConnectorVersion>8.0.16</mysqlConnectorVersion>
+        <mysqlConnectorVersion>8.0.25</mysqlConnectorVersion>
```

Comment:
Why not use 8.0.33 instead?

Reply:
8.0.33 is not compatible with Debezium we used in eip